modules/apps/commerce/commerce-order-web/src/main/resources/META-INF/resources/js/deleteCommerceOrdersPropsTransformer.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {fetch, openToast, postForm, sub} from 'frontend-js-web';
+
+export default function propsTransformer({
+	additionalProps: {namespace},
+	formName,
+	selectedItemsKey,
+	...otherProps
+}) {
+	const handleDelete = (
+		confirm,
+		{
+			actions: {
+				delete: {href},
```

Comment:
Are you destructuring the `href` here? If so, it should be done above the `handleDelete` definition, this is not very readable code.

Reply:
Yes, I agree. I was destructuring both `href` and `id` and it was unreadable. But I cannot do it above `handleDelete`, it needs to be inside this function because the object is passed by event. I went for good old dot notation, it looks better to me. [Check it out](https://github.com/liferay-commerce/liferay-portal/pull/3560/commits/56a63d84db78f2553b0c68403f342576753210f1#diff-65ced4c526ecf3453f9b03f6b20459ee50d0ede9c8c4be19821725e456e95e97R29-R33).